packages/metrics/src/features/locationStatistics/service.ts

```diff
@@ -90,12 +89,8 @@ async function cacheOfficeCount(authHeader: IAuthHeader) {
     .forEach((location) => dfs(locationsMap, location, adjacency))
 }
 
-function isOffice(location: Location) {
-  return (
-    location.type?.coding?.find(
-      ({ system }) => system === `${OPENCRVS_SPECIFICATION_URL}location-type`
-    )?.code === 'CRVS_OFFICE'
-  )
+function isState(location: Location) {
+  return location.identifier?.find((district) => district.value === 'STATE')
```

Comment:
```suggestion
  return location.identifier?.find(({ value }) => value === 'STATE')
```
The identifiers aren't really districts so let's not name them as such